index.js

```diff
@@ -124,30 +218,27 @@ try {
         )
         .click();
       console.log('INFO: Never-Consent enabled for all pages.');
-
-      fs.writeFileSync('profiles/withGhostery/onboarded', '');
     }
+
+    fs.writeFileSync(`profiles/withGhostery/${selectedBrowser}/onboarded`, '');
   }
 
-  console.log(`INFO: Open websites from for region: ${region}.`);
+  console.info(`INFO: Open websites from for region: ${region}.`);
 
   if (isGhosteryEnabled) {
     // Wait for Ghostery extension to download fresh Ad-blocking filters
     await sleep(1000 * 20);
   }
 
+  await driver.executeScript('window.open()', '');
+  const handle = await driver.getAllWindowHandles();
+  await driver.switchTo().window(handle[0]);
+  await driver.close();
+  await driver.switchTo().window(handle[1]);
```

Comment:
what does this do? and why is it needed now?

Reply:
It was here before - to switch to a new tab which was opened because of the onboarding and to close the empty tab.

Reply:
Would be worth leaving a comment in the code. I also struggle with it. As I understand, the purpose is to ensure the onboarding is not opening during the benchmarks?